Objective-C/UserSampleId/Podfile

```diff
@@ -1,8 +1,3 @@
-# Uncomment this line to define a global platform for your project
-# platform :ios, '8.0'
-# Uncomment this line if you're using Swift
-# use_frameworks!
-
 target 'UserSampleId' do
-  pod 'SwrveSDK'
+  pod 'SwrveSDK', '4.8.0'
```

Comment:
It possibly should. It installs 4.8.0 either way. 
Made the change so at least the version number matches our release notes